### ISSUE TITLE
Netlify: PHP Fatal error:  require(): Failed opening required '/opt/build/repo/vendor/autoload.php' (include_path='.:/usr/share/php') in /opt/build/repo/please on line 18

### ISSUE THREAD

**Issue by devfaysal** [statamic/ssg#97]
php please ssg:generate command working fine locally but failing in netlify deploy with the errors below. I was using Statamic 3.0 and it was working fine, but today I upgraded to 3.3 and getting errors.

PHP Warning:  require(/opt/build/repo/vendor/autoload.php): failed to open stream: No such file or directory in /opt/build/repo/please on line 18
PHP Fatal error:  require(): Failed opening required '/opt/build/repo/vendor/autoload.php' (include_path='.:/usr/share/php') in /opt/build/repo/please on line 18

"statamic/cms": "^3.3",
"statamic/ssg": "^1.0",

**Comment by duncanmcclean** [comment]
You need to run ‘composer install’ on Netlify before you run it.

**Comment by devfaysal** [comment]
> 

No, Netlify itself Run `composer install` if it is a PHP project.

I fixed the problem, but it was a weird issue.
I deleted everything from the database/factories folder as these were useless when I started using Statamic. 'composer install' command failed because of that. I just went through the full deploy log, then I noticed this.

![Screenshot(598)](https://user-images.githubusercontent.com/16212149/169686107-6f91d448-4c36-4cac-88a6-e552e1f15c90.png)

Now I created a factory and pushed it, the Build is completed successfully.
